domains/anees.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "email": "aneesnizam2552@example.com"
+  },
+  "record": {
```

it should be ```"records"``` with "s", https://docs.is-a.dev/domain-structure/#-records-required

```suggestion
  "records": {
```